ex03.ex

```diff
@@ -105,9 +124,20 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal( []    , []     ), do: true 
```

Comment:
Again, too much logic, and not enough pattern matching  :)  (no deduction though)

How about:

~~~ elixir
  def list_equal([], []),               do: true
  def list_equal([ h | t1 ], [h | t2]), do: list_equal(t1, t2)
  def list_equal(_, _),                 do: false
~~~